draft-ietf-rats-corim.md

```diff
@@ -1994,11 +1995,27 @@ If any codepoint present in the stateful environment `measurement-values-map` do
 If all checks above have been performed successfully then the stateful environment matches.
 If none of the candidate entries match the stateful environment entry then the stateful environment does not match.
 
-## Verifier Augmentation (Phase 5) {#sec-phase5}
-
-## Policy Augmentation (Phase 6) {#sec-phase6}
-
-## Attestation Results Production and Transformationn (Phase 7) {#sec-phase7}
+## Phases 5, 6, and 7 {#sec-phases567}
+
+Phase 5 is optional depending on implementation design. Verifier implementation of consistency, integrity, or validity checks could be represented as Claims that augment the ACS.
+Such Claims originate from the Verifier and are asserted under the authority of the Verifier.
+For example, the `attest-key-triple-record` may contain a cryptographic key to which the Verifier applies certificate path construction and verification. The verification result may reveal an expired certificate.
+The Verifier implementation might generate a certificate path validation exception that is handled externally, or it could generate a Claim that the certificate path isn't valid.
+
+Phase 6 is optional depending on implementation design as external representations of appraisal policy is out-of-scope for this document.
+Appraisal policy inputs could result in Claims that augment the ACS.
+For example, an Appraisel Policy for Evidence may specify that if all of a collection of subcomponents satisfy a particular quality metric, the top-level component also satisfies the quality metric.
+The Verifier might generate an Endorsement ECT for the top-level component that asserts a quality metric.
+Details about the policy applied may also augment the ACS.
+An internal representation of policy detials, based on the policy ECT as described in {{sec-ir-policy}}, contain the environments affected by the policy with policy identifers as Claims.
+
+Phase 7 is optional depending on implementation design.
+Attestation Results rely on input from the ACS, but may not be interested in actual ACS entries.
+For example, Attestation Results processing may map the ACS state to a generalized trustworthiness state such as {{-ar4si}}.
+Generated Attestation Results Claims may be specific to a particular Relying Party.
+Hence, the Verifier may need to maintain multiple Attestation Results contexts.
+An internal representation of Attestation Results as separate contexts, (see {{sec-ir-ars}}), ensures Relying Party specific processing doesn't modify the ACS, which is common to all Relying Parties.
```

Comment:
```suggestion
An internal representation of Attestation Results as separate contexts (see {{sec-ir-ars}}) ensures Relying Party–specific processing does not modify the ACS, which is common to all Relying Parties.
```
Parentheticals aren't also appositives.

I don't see how the sec-ir-ars reference elaborates the description of [ + ECT ] as "separate contexts". It's just another data definition that's equivalent to an ACS.

I don't think we've specified that the ACS is common to all relying parties either. There's nothing about the data definition that "ensures processing doesn't modify" something else. It's an organization technique that may help with that, but I don't know what this text is trying to say.